### PR TITLE
refined object storage procedure

### PR DESCRIPTION
**Pull request by klgill**
https://issues.redhat.com/browse/RHOSPDOC-1655

### CODE REVIEW

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 1]
```diff
@@ -1,26 +1,25 @@
 [id="adopting-the-object-storage-service_{context}"]
 
-= Adopting the Object Storage service
+= Adopting the {object_storage}
 
-This section only applies if you are using OpenStack Swift as {object_storage_first_ref}. If you are using the Object Storage API of Ceph RGW this section can be skipped.
+If you are using OpenStack Swift as the {object_storage_first_ref}, adopt the {object_storage} to the {rhos_long} environment. If you are using the Object Storage API of the Ceph Object Gateway (RGW), skip this procedure.
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:5]: @cschwede What is meant by, “If you are using OpenStack Swift as an Object Storage service”? 
Can we say, “If you are using Object Storage as a service" instead?

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
 
 .Prerequisites
 
-* Previous adoption steps completed.
-* The {object_storage} storage backend services must still be running.
-* Storage network has been properly configured on the {OpenShift} cluster.
+* Ensure that the {object_storage} storage back-end services are running.
+* Ensure that the storage network is properly configured on the {OpenShift} cluster.
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:10]: @cschwede How would customers ensure that their storage network is configured prooperly?
One suggestion is to link to the following guide, if applicable: https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/configuring_storage/index

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 3]
```diff
@@ -49,11 +48,11 @@ binaryData:
 EOF
 ----
 
-. Patch `OpenStackControlPlane` to deploy the {object_storage}:
+. Patch the `OpenStackControlPlane` custom resource to deploy the {object_storage}:
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:51]: @cschwede Which file is being updated in this step? Is it the openstack_control_plane.yaml file, which defines the OpenStackControlPlane CR?

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 4]
```diff
@@ -132,5 +131,4 @@ Hello World!
 ----
 
 [NOTE]
-At this point data is still stored on the previously existing nodes. For more information about migrating the actual data from the old
-to the new deployment, see xref:migrating-object-storage-data-to-rhoso-nodes_migrate-object-storage-service[Migrating the {object_storage_first_ref} data from {OpenStackShort} to {rhos_long} nodes].
+The Object Storage data is still stored on the existing {OpenStackShort} nodes. For more information about migrating the actual data from the {OpenStackShort} deployment to the {rhos_long} deployment, see xref:migrating-object-storage-data-to-rhoso-nodes_migrate-object-storage-service[Migrating the {object_storage_first_ref} data from {OpenStackShort} to {rhos_long} nodes].
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:134]: @cschwede Please confirm that my edits to this note are accurate, particularly the first sentence.

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc (hunk 3 above)
Comment (cschwede) [reply docs_user/modules/proc_adopting-the-object-storage-service.adoc:51]: Yes, exactly that file. One can either patch an already existing CR (like in this example case) or patch the file before creating the CR.

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc (hunk 4 above)
Comment (cschwede) [reply docs_user/modules/proc_adopting-the-object-storage-service.adoc:134]: LGTM.

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc (hunk 2 above)
Comment (cschwede) [reply docs_user/modules/proc_adopting-the-object-storage-service.adoc:10]: I propose to link here to [https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-rhocp-for-rhoso#proc_configuring-the-data-plane-network_preparing](https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-rhocp-for-rhoso#proc_configuring-the-data-plane-network_preparing)

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 5]
```diff
@@ -1,26 +1,25 @@
 [id="adopting-the-object-storage-service_{context}"]
 
-= Adopting the Object Storage service
+= Adopting the {object_storage}
 
-This section only applies if you are using OpenStack Swift as {object_storage_first_ref}. If you are using the Object Storage API of Ceph RGW this section can be skipped.
+If you are using OpenStack Swift as an {object_storage}, adopt the {object_storage} to the {rhos_long} environment. If you are using the Object Storage API of the Ceph Object Gateway (RGW), skip this procedure.
```
Comment (mickogeary) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:5]: You could maybe put "Optional" at the start of this sentence. Is the point of this line that you have to adopt the {object_storage} to the {rhos_long} environment if you use OpenStack Swift as an {object_storage}? If so, maybe: "If you use OpenStack Swift as an {object_storage}, ensure that you adopt the {object_storage} to the {rhos_long} environment."

Skip what procedure? The "adopt the {object_storage} to the {rhos_long} environment" or what comes next? The original draft makes it look like it's what follows, whereas the rewrite could be interpreted as the step you just mentioned in the previous line. Could clarify with "skip the following procedure" or "you do not need to adopt the {object_storage} to the {rhos_long} environment." or whatever the case may be.

Should you say the OpenStack Object Store Service (Swift) or something like that? 

Also maybe OpenStack could be made into a variable so it is OpenStack upstream and RHOSP downstream

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 6]
```diff
@@ -1,26 +1,25 @@
...
 
 .Prerequisites
 
-* Previous adoption steps completed.
-* The {object_storage} storage backend services must still be running.
-* Storage network has been properly configured on the {OpenShift} cluster.
+* Ensure that the {object_storage} storage back-end services are running.
+* Ensure that the storage network is properly configured on the {OpenShift} cluster. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-rhocp-for-rhoso#proc_configuring-the-data-plane-network_preparing[Configuring the data plane network] in _Deploying Red Hat OpenStack Services on OpenShift_.
 
 //No new environmental variables need to be defined, though you use the
```
Comment (mickogeary) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:12]: you can delete these hidden comments if you don't need them anymore

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 7]
```diff
@@ -90,13 +89,13 @@ spec:
 
 .Verification
 
-* Inspect the resulting {object_storage} service pods:
```
Comment (mickogeary) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:93]: does {object_storage} variable already contain the word service?

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc [hunk 8]
```diff
@@ -90,13 +89,13 @@ spec:
...
+* Inspect the resulting {object_storage} pods:
 +
 ----
 $ oc get pods -l component=swift-proxy
 ----
 
-* Check that the {object_storage} proxy service is registered in the {identity_service}:
```
Comment (mickogeary) [inline docs_user/modules/proc_adopting-the-object-storage-service.adoc:99]: "Check" looks ok to me, could also say "check if" or "verify"

File: docs_user/modules/proc_adopting-the-object-storage-service.adoc (hunk 7 above)
Comment (klgill) [reply docs_user/modules/proc_adopting-the-object-storage-service.adoc:93]: Yes.

**Pull request merged**